sdk/storage/azure-storage-common/src/test-shared/java/com/azure/storage/common/test/shared/extensions/TestHeaderExtension.java

```diff
@@ -19,14 +19,17 @@ public void visitSpec(SpecInfo specInfo) {
 
     private static class TestHeaderIterationInterceptor implements IMethodInterceptor {
         @Override
-        public void intercept(IMethodInvocation invocation) {
+        public void intercept(IMethodInvocation invocation) throws Throwable {
             // Print out the test name to create breadcrumbs in our test logging in case anything hangs.
             String testName = TestNameProvider.getTestName(invocation.getIteration());
-            System.out.printf("========================= %s =========================%n", testName);
+            System.out.printf("%s is starting", testName);
```

Comment:
Thoughts on switching over to using a logger here given that CI has logging configurations?

Also, add a `%n` to the `printf` so it is new-lined.

Reply:
I'm going to add logger but keep printf - printf output displays on devops job page which is nice, but having entries in log file are going to help analyze it without crossreferencing with devops output.